plugins/in_http/http_prot.c

```diff
@@ -523,7 +523,7 @@ static int process_payload(struct flb_http *ctx, struct http_conn *conn,
         return -1;
     }
 
-    if (header->val.len == 16 &&
+    if (header->val.len >= 16 &&
```

Comment:
According to [RFC 2616](https://datatracker.ietf.org/doc/html/rfc2616#section-3.7), media types can have parameters which should be separated from the main type and subtype with a colon (and I think optionally a space) which means, in the case of a value longer than 16 characters you would have to verify that the seventeenth character is either a space a colon in order to avoid matching values like "application/json_but_with_some_twist_that_we_do_not_want_to_match".

Reply:
@leonardo-albertovich Did you mean semi-colon? `;`?